back/lib/tasks/single_use/20251029_delete_legacy_file_records.rake

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+namespace :single_use do
+  desc 'Delete all legacy file records after migration to new Files system'
+  # This rake task deletes records explicitly using +destroy!+ before we delete the tables
+  # in a migration in order to run the destroy callbacks and clean up associated files on
+  # S3.
+  task delete_legacy_file_records: [:environment] do |_t, _args|
+    SemanticLogger.named_tagged(
+      rake_task: 'single_use:delete_legacy_file_records',
+      run_id: SecureRandom.uuid
+    ) do
+      legacy_classes = [
+        IdeaFile,
+        ProjectFile,
+        EventFile,
+        PhaseFile,
+        StaticPageFile,
+        ProjectFolders::File
+      ]
+
+      # region stats_and_logging
```

Comment:
Not used this comment style before, but these are just for folding the code in the IDE?

Reply:
Yeah, I was just playing around with it to hide some of the cluttered code that wasn’t really part of the main logic. It’s probably not something I’d use in the core codebase, but it did make the code easier to read in the IDE.

<img width="643" height="796" alt="SCR-20251031-ivgy" src="https://github.com/user-attachments/assets/896bef4a-35d4-410d-8f09-3636ffc32e26" />